openwebstart/src/main/java/com/openwebstart/os/ScriptFactory.java

```diff
@@ -41,6 +39,18 @@ public static String createStartScriptForMac(final JNLPFile jnlpFile) {
         return SCRIPT_START + System.lineSeparator() + "open -a " + executable + " " + jnlpLocation;
     }
 
+	public static String createSimpleStartScriptForMac(final JNLPFile jnlpFile) {
+		final URL srcUrl = jnlpFile.getSourceLocation();
+		final String schema = srcUrl.getProtocol();
+		final String url;
+		if ("http".equalsIgnoreCase(schema) || "https".equalsIgnoreCase(schema)) {
```

Comment:
please externalise "http", "https" and "jnlp" as constants. I assume that we already have defined them in a global constant class / interface but can not check at the moment.